src/components/Class/index.js

```diff
@@ -890,21 +873,33 @@ function Class({ classToEdit, indicator }) {
                   />
                 </>
               )}
-              {console.log(_.isEqual(prevClassState, formFieldsState))}
-              {console.log(prevClassState, formFieldsState)}
-              <button
-                type="submit"
-                className="submit"
-                disabled={loading && _.isEqual(prevClassState, formFieldsState)}
-              >
-                {loading ? (
-                  <Loader />
-                ) : isEditMode ? (
-                  "UPDATE CLASS"
-                ) : (
-                  "Create Class"
-                )}
-              </button>
+              {_.isEqual(initialFormState, formFieldsState) ? (
+                <div class="cursor">
+                  <button
```

Comment:
Don't render the button conditionally. Instead, set disable attribute conditionally. 
`  {const checkEquivalence = _.isEqual(initialFormState, formFieldsState)}
   <button
         disabled={checkEquivalence}
         className={checkEquivalence ? "pointer-disabled-class" : "pointer-enabled-class"}
`